compliance-report-imports/compliance-report-imports.cf

```diff
@@ -44,7 +44,8 @@ WHERE export_id IS null AND name IN ('OS is reported', 'Supported CentOS', 'Supp
          -X POST \
          --form 'data=<{{{.}}}' \
          --form 'public=true' \
-         --form 'overwrite=true'
+         --form 'overwrite=true' \
+         --form 'source=build module'
```

Comment:
What happens if it's run like this on an old version of Mission Portal?

I tested for myself, it still imports, so not a backwards incompatible change. +1